app/lib/actions/katello/host/update_deb_package_profile.rb

```diff
@@ -0,0 +1,55 @@
+module Actions
+  module Katello
+    module Host
+      class UpdateDebPackageProfile < Actions::EntryAction
```

Comment:
I'm not sure this class is going to be necessary if the other suggestion I made of the "combined profile" is used.

Reply:
Well the logic performed by that task must go somewhere.

Reply:
Most of the logic is performed in Actions::Katello::Host::UploadProfiles which this code duplicates. The unique aspects (creating the DB record) can remain here perhaps a class method would be suitable:
https://github.com/Katello/katello/blob/23f441f24884022fb9bebba410fd9cf6995a6fab/app/lib/actions/katello/host/upload_profiles.rb#L58